devtools/debug.py

```diff
@@ -225,4 +262,18 @@ def _process_args(self, ex: 'Any', args: 'Any', kwargs: 'Any') -> 'Generator[Deb
             yield self.output_class.arg_class(value, name=name, variable=kw_arg_names.get(name))
 
 
+def _make_call_context(call_frame: 'Optional[FrameType]', trace: bool) -> 'List[DebugFrame]':
```

Comment:
might make most sense for this to be part of `DebugFrame`

Reply:
There's not really much difference between a function and a static method, but all else being equal, I prefer functions.  I think they're easier to understand because you know they only have access to the public APIs of any classes they use.  But I can make it a method if you care strongly about it.